og.routing.yml

```diff
@@ -29,3 +29,83 @@ og.unsubscribe:
     parameters:
       group:
         type: entity:{entity_type_id}
+
+og.entity_autocomplete:
+  path: '/group/{entity_type_id}/{group}/autocomplete/{target_type}/{selection_handler}/{selection_settings_key}'
+  defaults:
+    _controller: '\Drupal\og\Controller\OgAutocompleteController:handleAutocomplete'
+  requirements:
+    _access: 'TRUE'
+  options:
+    parameters:
+      group:
+        type: entity:{entity_type_id}
+
+# OG Membership entity routes
+entity.og_membership.canonical:
+  path: 'group/{entity_type_id}/{group}/admin/members/{og_membership}'
+  defaults:
+    _title: 'Membership'
+    _entity_view: 'og_membership.full'
+  requirements:
+    _entity_access: 'og_membership.view'
```

Comment:
@jludwig 
If we allow access to this routing path, IMHO we need to provide something meaningful.
![membership-view](https://cloud.githubusercontent.com/assets/114076/23298278/5f2bf4de-fa7d-11e6-99a3-aac1be15cf6e.png)

Either we set it simply
```
_access: 'FALSE'
```
or we remove it if it's not a needed thing for anything, we should add basic information of the membership, like Member since (`created`) and `status`, otherwise this page looks like a stub.

Reply:
Seems to have been removed